BadHabbit/BadHabbit/Kit/DesignKit/Colors/Color.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Color.swift
+//  BadHabbit
+//
+//  Created by Viktoria Misiulia on 19/07/2023.
+//
+
+import UIKit
+
+extension UIColor {
+    // The CSS color value #CC6699 is written as 0xCC6699 in Swift’s hexadecimal number representation.
+    convenience init(hexNumber: UInt, alpha: CGFloat = 1.0) {
```

Comment:
For something like that there is a good practice in naming files:
- if your extension contain only one function or init we name file as _ExtendedClass+functionName_ in your example UIColor+hex
- If extension contains several functions we name due to function tasks, like UIColor+convenientInit

Reply:
Renamed to UIColor+hex